src/net.cpp

```diff
@@ -1862,6 +1862,12 @@ void CConnman::SetTryNewOutboundPeer(bool flag)
     LogPrint(BCLog::NET, "net: setting try another outbound peer=%s\n", flag ? "true" : "false");
 }
 
+void CConnman::StartExtraBlockRelayPeers()
```

Comment:
In e1437e1f5d3fb9ae95b1ca503893ff0ffb2e8c8d
> which allows dropping `#include <logging>` in the header file.

is this the only reason for this change? I'm not sure that doing this is required right now, and it seems to just be what necessitates the changes in the prior commit.

Reply:
No; see explanation in the first commit message.